source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/ray_caster/ray_caster_camera_cfg.py

```diff
@@ -44,6 +46,9 @@ class OffsetCfg:
     data_types: list[str] = ["distance_to_image_plane"]
     """List of sensor names/types to enable for the camera. Defaults to ["distance_to_image_plane"]."""
 
+    pattern_cfg: PinholeCameraPatternCfg = MISSING
```

Comment:
This is just to make it enforce the type of it?

Reply:
NIT: This might better be named pinhole_pattern_cfg unless we expect it to be a different type of pattern cfg

Reply:
`pattern_cfg` is the general term inherited from `RayCasterCfg`, if we rename it now, that would create an ambiguity